src/Components/Publishing/Display/Canvas/index.tsx

```diff
@@ -10,19 +11,32 @@ interface DisplayCanvasProps {
   campaign: any
 }
 
-export const DisplayCanvas: React.SFC<DisplayCanvasProps> = props => {
-  const { unit, campaign } = props
-  const disclaimer = <Disclaimer layout={unit.layout}>{unit.disclaimer}</Disclaimer>
+@track()
+export class DisplayCanvas extends React.Component<DisplayCanvasProps, any> {
+  @track(props => ({
+    action: "Click",
+    label: "Display ad carousel arrow",
+    entity_type: "display_ad",
+    campaign_name: props.campaign.name,
+    unit_layout: "canvas_slideshow"
+  }))
+  // tslint:disable-next-line:no-empty
+  componentDidMount() { }
 
-  return (
-    <DisplayContainer layout={unit.layout}>
-      <a href={unit.link.url} target="_blank">
-        <AdvertisementBy>{`Advertisement by ${campaign.name}`}</AdvertisementBy>
-      </a>
-      <CanvasContainer unit={unit} campaign={campaign} disclaimer={disclaimer} />
-      {unit.layout === "overlay" && disclaimer}
-    </DisplayContainer>
-  )
+  render() {
```

Comment:
Looks like this action already exists above in the slideshow component, is this supposed to be a link click for the slideshow instead?

Reply:
Good catch, this might have been a merge cleanup slip-up. This was an old attempt at [impression tracking](https://github.com/artsy/reaction/pull/314/files#diff-cbef750817163c85f91a3113eb1ca059R26). Removing.